packages/site/src/layout/SearchBox.tsx

```diff
@@ -43,53 +47,31 @@ export const SearchBox = ({
   const ref = useRef<InputTextRef>(null);
   const [search, setSearch] = useState("");
 
-  const filterFunction = (item: ContentListItem) =>
-    item.title.toLowerCase().includes(search.toLowerCase()) ||
-    item.additionalMatches?.find((e: string) =>
-      e.toLowerCase().includes(search.toLowerCase()),
+  const filterItems = (items: ContentListItem[], searchTerm: string) => {
+    const lowerCaseSearchTerm = searchTerm.toLowerCase();
+
+    return items.filter(
+      item =>
+        item.title.toLowerCase().includes(lowerCaseSearchTerm) ||
+        item.additionalMatches?.some((e: string) =>
```

Comment:
I feel like `.some` is all we need here. `.some` is just looking for if any `additionalMatches` match the search term and that's all we care about.
`.find` is going to return the matching element which we don't need in this case.